pkgs/top-level/all-packages.nix

```diff
@@ -3117,7 +3117,7 @@ with pkgs;
   };
 
   rocmPackages = rocmPackages_6;
-  rocmPackages_5 = recurseIntoAttrs (callPackage ../development/rocm-modules/5 { });
+  rocmPackages_5 = throw "ROCm 5 has been removed";
```

Comment:
Shouldn't this be in `aliases.nix` instead?

Reply:
I was not aware of that file, if you say so I will move it there.

Reply:
Ok, I did this now and it seems to make nix angry about a bunch of stuff that still depends on ROCm 5, which will take me some more time to look into.

Thanks for the hint!